docs/api/request.md

```diff
@@ -129,6 +145,10 @@ to PHP's `$_FILES` superglobal.
 Each file in this array implements the `Psr\Http\Message\UploadedFileInterface`:
 
 ```php
+<?php
+
+// …
+
```

Comment:
I think this one slipped under your radar ^^

Reply:
This example intentionally omits the `filename` attribute as the code is not part of a request handler function. I agree it might be a good idea to clean this example up in a separate follow-up PR some time in the future :+1:

Reply:
Seems reasonable, you got my approval :+1: